### ISSUE TITLE
Policeman taking a lot of FireFox' Memory. Is there a way to "reset"?

### ISSUE THREAD

**Issue by gitgetgot** [futpib/policeman#235]
Atm, the policeman takes a lot of RAM, making FF (43.x on Win7 64Bit, 22GB RAM) very hard to use (long lags in UI)
![screenclip](https://cloud.githubusercontent.com/assets/974980/13466870/2d522240-e09c-11e5-8d53-e4d47aff7958.png)

Is there a way to "reset" or "defrag" or "clean" the momory policeman s using? Beside restart FF...

Like i.e. forget about all the rules for closed tabs (I've got 40 open Tabs, 20 active ,20 not loaded/inactive) 
or some kind of restriction ?

This happens mostly, when having FF open for a couple of days.

I tested this with policeman deactivated and it FF is not lagging (that much ;-) )

**Comment by andrew-phi** [comment]
Yes, slowdown is noticable. Even while it saves time and traffic by blocking resources, it slows FF down by itself somehow.
